cmd/spire-server/cli/run/run.go

```diff
@@ -460,12 +460,8 @@ func NewServerConfig(c *Config, logOptions []log.Option, allowUnknownConfig bool
 
 	if !hasExpectedTTLs(sc.CATTL, sc.SVIDTTL) {
 		sc.Log.Warnf("The default_svid_ttl is too high for the configured ca_ttl value. SVIDs with shorter lifetimes may be issued. "+
-			"Please set default_svid_ttl to %v or less, or ca_ttl to %v or more, to guarantee the default_svid_ttl lifetime.",
-			calcSVIDTTL(sc.CATTL), calcCATTL(sc.SVIDTTL))
-	}
-
-	if sc.SVIDTTL > ca.ActivationThresholdCap {
-		sc.Log.Warn("The SVID TTL cannot be guaranteed in all cass - SVIDs with shorter TTLs may be issued if the signing key is expiring soon")
```

Comment:
May I ask why this is deleted?

Reply:
Heh I figured I might have missed something with this log line which is why I asked you to take a look! Thanks for pointing it out :)

I could not see a difference in the condition under which these lines are logged. It seemed to me that, if one line is logged then so is the other. Maybe I misread it? It felt redundant to me. Are we trying to say two different things?

Reply:
In my understand, If ca_ttl is a large value, the ca manager try to activate it 7 days before the expiration instead of 1/2.

https://github.com/spiffe/spire/blob/345cdf22a5c68be3b01e15643844aa2dc26295b9/pkg/server/ca/manager.go#L1061-L1068

Therefore, the SVID may be issued with a TTL shorter than the configured value at some case.
(e.g. ca_ttl=1152h , svid_ttl=192h)

https://github.com/spiffe/spire/blob/345cdf22a5c68be3b01e15643844aa2dc26295b9/pkg/server/ca/ca.go#L249
https://github.com/spiffe/spire/blob/345cdf22a5c68be3b01e15643844aa2dc26295b9/pkg/server/ca/ca.go#L318-L326

Reply:
how about this?

```go
if !hasExpectedTTLs(sc.CATTL, sc.SVIDTTL) {
		sc.Log.Warnf("The default_svid_ttl is too high for the configured ca_ttl value. ...)
} else if sc.SVIDTTL > ca.ActivationThresholdCap {
		sc.Log.Warn("The SVID TTL cannot be guaranteed in all cass ...")
}
```

Reply:
@evan2645 
How do you feel about this please?